pkg/cmd/pr/pr.go

```diff
@@ -308,6 +332,23 @@ func (o *Options) Validate() error {
 		o.GitCommitUsername = "jenkins-x-bot"
 	}
 
+	if o.PipelineCommitSha == "" {
+		o.PipelineCommitSha = os.Getenv("PULL_PULL_SHA")
```

Comment:
Unfortunately I don't think this env var is actually populated in the release pipeline :S